src/viking.js

```diff
@@ -1,11 +1,105 @@
 // Soldier
```

Comment:
**A general tip**: The formatting of your document can be made easier if you install & use the extension "Prettier" for VsCode. Search for "Prettier" in your extension marketplace on vscode or find it here --> https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode 

You can also find how you install and use Prettier here --> https://www.youtube.com/watch?v=h3PJjP0nE98&ab_channel=JamesQQuick